botocore/client.py

```diff
@@ -239,16 +162,16 @@ def _get_client_args(self, service_model, region_name, is_secure,
             if client_config.user_agent_extra is not None:
                 user_agent += ' %s' % client_config.user_agent_extra
 
-        signer = RequestSigner(service_model.service_name, region_name,
-                               service_model.signing_name,
-                               signature_version, credentials,
-                               event_emitter)
+        signer = RequestSigner(
+            service_name, endpoint_config['signing_region'], service_name,
```

Comment:
Looks like signing_name isn't being used anymore.  How do we account for this now?